config/secrets.yml

```diff
@@ -14,7 +14,7 @@ development:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
 
 test:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: 7053b205c9559f83960f18d23f10d36cacd737c0e17e0dd71cbb90d2f3e6
```

Comment:
> isn't this generated by template?

@mfojtik you type quicker than I :smile: 

It is exposed in the template and autogenerated by default, and used to populate an env var in the deployment config.

So the config is/was effectively empty when the build hook runs.